packages/docs/README.md

```diff
@@ -39,9 +39,9 @@ This will serve the wesite locally in your computer at `http://localhost:3000/`.
 
 ## API Reference generation
```

Comment:
This description is not entirely accurate, but if it's okay I can update it next week.